RATIONALE.md

```diff
@@ -592,8 +594,9 @@ not used again after initialization.
 wazero currently supports only one pre-opened file, "/" and so that is the name
 returned by `fd_prestat_dir_name` for file descriptor 3 (STDERR+1).
 
-See https://github.com/WebAssembly/wasi-libc/blob/a02298043ff551ce1157bc2ee7ab74c3bffe7144/libc-bottom-half/sources/preopens.c
-See https://github.com/ziglang/zig/blob/master/lib/std/fs/wasi.zig#L50-L53
+See
+ * https://github.com/WebAssembly/wasi-libc/blob/a02298043ff551ce1157bc2ee7ab74c3bffe7144/libc-bottom-half/sources/preopens.c
+ * https://github.com/ziglang/zig/blob/master/lib/std/fs/wasi.zig#L50-L53
```

Comment:
master will change really fast, so permalink would be better

Reply:
true!